README.md

```diff
@@ -2,157 +2,136 @@
 page_type: sample
 languages:
 - javascript
+- nodejs
 products:
-- azure
-description: "For Service-to-Azure-Service authentication, the approach so far involved creating an Azure AD application and associated credential,"
-urlFragment: app-service-msi-keyvault-node
+- azure-key-vault
+- azure-app-service
+description: "How to set and get secrets from Azure Key Vault with Azure Managed Identities and Node.js."
+urlFragment: get-set-keyvault-secrets-managed-id-nodejs
 ---
 
-# Use Key Vault from App Service with Managed Service Identity and Nodejs
+# How to set and get secrets from Azure Key Vault with Azure Managed Identities and Node.js
 
-## Background
```

Comment:
Can you please include some of this context at the beginning, but not all of it.  It will need to be re-written and shortened, but a short blurb describing what to expect in the sample would be good.